appimagebuilder/modules/setup/generator.py

```diff
@@ -108,7 +108,7 @@ def _patch_interpreted_executables(self, executables, patcher: ExecutablesPatche
         for executable in interpreted_executables:
             allowed = True
```

Comment:
I guess that it would be better to put tho whole check in a function so it's easier to read

Reply:
Done!